src/screens/Wallet.js

```diff
@@ -269,6 +269,7 @@ function Wallet() {
     context.showModal(MODAL_TYPES.LEDGER_SIGN_TOKEN, {
       token,
       modalId: 'signTokenDataModal',
+      cb: (newValue) => console.log(`New token signature state: ${newValue}`)
```

Comment:
The `cb` parameter is mandatory for this modal, and was causing a crash because it wasn't informed. However, it has no application for this screen, so it's only used as a dummy output.

Reply:
Ok, but do we really need this console.log here?

Reply:
We don't! Replaced it with a dummy function and an explanatory comment on https://github.com/HathorNetwork/hathor-wallet/pull/546/commits/a1f9556eaa907d5895647f4c4f2ca0476836224b